app/bundle/src/main/scala/org/bitcoins/bundle/gui/NeutrinoConfigPane.scala

```diff
@@ -76,12 +97,18 @@ class NeutrinoConfigPane(
   }
 
   def getConfig: Config = {
-    val configStr =
+    var configStr =
       s"""
+         |bitcoin-s.network = ${networkComboBox.value.value}
```

Comment:
you are gonna wanna call `DatadirUtil.networkStrToDirName` on this value so we use the correct folder.